docs/sql-programming-guide.md

```diff
@@ -1489,7 +1489,7 @@ See the [Apache Avro Data Source Guide](avro-data-source-guide.html).
 
  * The JDBC driver class must be visible to the primordial class loader on the client session and on all executors. This is because Java's DriverManager class does a security check that results in it ignoring all drivers not visible to the primordial class loader when one goes to open a connection. One convenient way to do this is to modify compute_classpath.sh on all worker nodes to include your driver JARs.
  * Some databases, such as H2, convert all names to upper case. You'll need to use upper case to refer to those names in Spark SQL.
-
+ * Users can specify vendor-specific JDBC connection properties in the data source options to do special treatment. For example, `spark.read.format("jdbc").option("url", oracleJdbcUrl).option("oracle.jdbc.mapDateToTimestamp", "false")`. `oracle.jdbc.mapDateToTimestamp` defaults to true, users often need to disable this flag to avoid Oracle date being resolved as timestamp.
```

Comment:
Ur.. I see. This section is troubleshooting, so I think you'd be better to illustrate a concrete trouble case, then you describe a solution for that.

Reply:
This looks fine to me. 

@maropu Your idea is great! We need more examples in the troubleshooting. Currently, our document for Spark SQL and Core needs a major update.  Maybe we can do it after we finish the reorg of the documentation. https://issues.apache.org/jira/browse/SPARK-24499?

Reply:
@maropu @gatorsmile Could you please suggest something else to do here?